contracts/NodeDelegatorLST.sol

```diff
@@ -10,35 +11,31 @@ import { LRTConstants } from "./utils/LRTConstants.sol";
 import { LRTConfigRoleChecker, ILRTConfig } from "./utils/LRTConfigRoleChecker.sol";
 
 import { IDelegationManager } from "./eigen/interfaces/IDelegationManager.sol";
-import { IEigenPodManager, IEigenPod } from "./eigen/interfaces/IEigenPodManager.sol";
 import { ISignatureUtils } from "./eigen/interfaces/ISignatureUtils.sol";
 import { IStrategyManager, IStrategy } from "./eigen/interfaces/IStrategyManager.sol";
-import { INodeDelegator } from "./interfaces/INodeDelegator.sol";
-import { ISSVNetwork, Cluster } from "./interfaces/ISSVNetwork.sol";
+import { INodeDelegatorLST } from "./interfaces/INodeDelegatorLST.sol";
 import { IOETH } from "./interfaces/IOETH.sol";
 import { IWETH } from "./interfaces/IWETH.sol";
 
-struct ValidatorStakeData {
-    bytes pubkey;
-    bytes signature;
-    bytes32 depositDataRoot;
-}
-
-/// @title NodeDelegator Contract
+/// @title NodeDelegatorLST Contract
 /// @notice The contract that handles the depositing of assets into strategies
-contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradeable, ReentrancyGuardUpgradeable {
+contract NodeDelegatorLST is
+    INodeDelegatorLST,
+    LRTConfigRoleChecker,
+    PausableUpgradeable,
+    ReentrancyGuardUpgradeable
+{
     address public constant DEAD_ADDRESS = 0x000000000000000000000000000000000000dEaD;
     /// @dev The Wrapped ETH (WETH) contract address with interface IWETH
     address public immutable WETH;
 
     /// @dev The EigenPod is created and owned by this contract
-    address public eigenPod;
+    address public deprecated_eigenPod;
     /// @dev Tracks the balance staked to validators and has yet to have the credentials verified with EigenLayer.
     /// call verifyWithdrawalCredentials to verify the validator credentials on EigenLayer
-    uint256 public stakedButNotVerifiedEth;
+    uint256 public deprecated_stakedButNotVerifiedEth;
 
     uint256 internal constant DUST_AMOUNT = 10;
-    mapping(bytes32 pubkeyHash => bool hasStaked) public validatorsStaked;
```

Comment:
🟥 Removing this mapping will cause the later withdraw mappings to move up, thus messing up the storage ids. I think in the best case this would make us lose track of active public and internal withdraws and then write any new ones to the wrong storage slots?